app/vmselect/prometheus/prometheus.go

```diff
@@ -1133,3 +1142,27 @@ func getCommonParams(r *http.Request, startTime time.Time, requireNonEmptyMatch
 	}
 	return cp, nil
 }
+
+func errWithContext(err error, context string) error {
+	if err == nil {
+		return nil
+	}
+	str := err.Error()
+	if !strings.Contains(str, storage.ErrSeriesLimit.Error()) {
+		return err
+	}
+
+	str = strings.Replace(str, storage.FlagSeriesLimit, context, -1)
```

Comment:
Remove the level of indirection from here in order to improve code readability, e.g. substitute `storage.FlagSeriesLimit` with string literal.